util.py

```diff
@@ -279,3 +279,11 @@ def check_file_sha256_checksum(filename, reference_checksum):
     :param str filename: a single file to be checked
     """
     assert compute_file_sha256_checksum(filename) == reference_checksum
+
+
+def resolve_sis_paths_in_dict(dictionary):
```

Comment:
```suggestion
def resolve_sis_paths_in_dict(dictionary):
    """
    Iterates through a (nested) dict and replaces all Path instances 
    with the respective file paths. Only to be used in the Worker.
    
    :param dict dictionary: a dict containing Path objects to be replaced
    """
```
@JackTemaki you like the docstring?

Reply:
> Only to be used in the Worker

Perfect!